doc/usermanual/lighttable/concepts/undoredo.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE sect2 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+               "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+		<!ENTITY % darktable_dtd SYSTEM "../../dtd/darktable.dtd">
+		%darktable_dtd;
+		]>
+<sect2 status="final" id="lt_undo_redo">
+
+  <title>Undo/redo</title>
+
+    <indexterm>
+      <primary>undo</primary>
+    </indexterm>
+
+    <indexterm>
+      <primary>redo</primary>
+    </indexterm>
+
+    <para>
+      Most changes in the lighttable are recorded and can be
+      reverted. This includes modifications in color labels, ratings,
+      geo-localization, tags, metadata, copy/paste of history or
+      applying a style. Thanks to this recording it is possible to
+      undo changes or redo them to recover a previous state. Note that
+      this undo/redo facility is unlimited in the number of steps
+      while in the lighttable but it is reset each time the view is
+      switched (going to the darkroom for example).
+    </para>
+
+    <para>
+      The default key shortcuts are:
+      <itemizedlist>
+
+        <listitem><para>
+          <emphasis>Ctrl-z</emphasis> : undo last modifications
```

Comment:
nitpick probably to be ignored, if only for consistency with the rest of the doc: the standard way of tagging that sort of things in DocBook would be <keycombo><keycap>Ctrl</keycap><keycap>Z</keycap></keycombo>.

Reply:
Indeed for consistency we need to keep ctrl-z.

Reply:
.... github ate my exact DocBook tagging. Let my try again:
`<keycombo><keycap>Ctrl</keycap><keycap>Z</keycap></keycombo>`.
Anyway. As I was saying, nitpicking :P